### DISCUSSION TITLE
От чего зависит скорость передачи данных?

### DISCUSSION THREAD

**Discussion by juPzHqLBa** [localsend/localsend#1389] [L-Русский]
Приветствую. Хотелось бы узнать, от чего зависит скорость передачи данных? У меня она в среднем 5.5 MB/s (Wi-Fi 5, частота 2.4 GHz)

**Comment by KhamzaGord** [comment]
(от типа сооеденения и от использования впн прокси или запретов ) - Мое мнение !

**Reply by KostZen** [reply]
А как ее увеличить? Нигде нет инфы

**Reply by KostZen** [reply]
<img width="151" height="61" alt="image" src="https://github.com/user-attachments/assets/824df6f9-6493-4c7f-b4c9-2627f5c6548c" />
это будет вечность))
